### ISSUE TITLE
Autofill preview tooltip shown in the wrong location if the document have frozen row/columns 

### ISSUE THREAD

**Issue by bayramcicek** [CollaboraOnline/online#10253]
### Describe the Bug
please see: https://github.com/CollaboraOnline/online/pull/10037#issuecomment-2411599872

### Steps to Reproduce
- freeze a row/column -> scroll the unfrozen area a bit -> use autofill in the frozen area -> tooltip shown in the wrong location
  - core side thinks upper side(frozen side) is also scrolled(moved up) -> possibly...

### Screenshots
https://github.com/user-attachments/assets/89757f60-9368-4a3e-b4a9-818680963e50

### Desktop
COOLWSD version: 24.04.9.0(git hash: becc2356)
LOKit version: Collabora Office 24.04.9.0(git hash: bc4a9f0)
Served by: Manjaro Linux
Server ID: 1af385c5

**Comment by pedropintosilva** [comment]
I believe this is not really related with autofill but with the selection and and the scrolling. In core when dragging a selection that started bellow a freeze pane and keeps going further than the split pane line it automatically scrolls the view. That doesn't happen on Online. It seems it was what @bayramcicek was hinting at in https://github.com/CollaboraOnline/online/pull/10037#issuecomment-2411599872

**Comment by bayramcicek** [comment]
> I believe this is not really related with autofill but with the selection and and the scrolling. In core when dragging a selection that started bellow a freeze pane and keeps going further than the split pane line it automatically scrolls the view. That doesn't happen on Online. It seems it was what @bayramcicek was hinting at in [#10037 (comment)](https://github.com/CollaboraOnline/online/pull/10037#issuecomment-2411599872)

Yes I agree. I'll edit the description and add there screen recordings of core vs online. Thanks

**Comment by bayramcicek** [comment]
> > I believe this is not really related with autofill but with the selection and and the scrolling. In core when dragging a selection that started bellow a freeze pane and keeps going further than the split pane line it automatically scrolls the view. That doesn't happen on Online. It seems it was what @bayramcicek was hinting at in [#10037 (comment)](https://github.com/CollaboraOnline/online/pull/10037#issuecomment-2411599872)
> 
> Yes I agree. I'll edit the description and add there screen recordings of core vs online. Thanks

Hmm. So, let's make things clear here. I see we have 2 issues here under the `Freezing row/columns` section here https://github.com/CollaboraOnline/online/pull/10037#issuecomment-2411599872.

- the "scrolling" bug -that we are discussing here ^ - refers to the 1st issue.
- and the bug on this page, refers to the 2nd issue.

Therefore, instead of editing the title & desc. of this bug; we need to open a separate ticket for the 1st issue:
- https://github.com/CollaboraOnline/online/issues/10269

**Comment by smehrbrodt** [comment]
Fixed with https://gerrit.collaboraoffice.com/plugins/gitiles/online/+/90369c0de2be7de19ce9be1012e3f364cd802eaa%5E%21/
